solutions/remotemonitoring/single-vm/setup-wrapper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+
+# Solution: remote monitoring
+
+# Important:
+# 1. The script runs in an environment with old/strict shell support, so don't rely on Bash niceties
+# 2. The script is designed NOT to throw errors, to avoid secrets ending in azureiotsolutions.com logs
+# 3. In case of errors, the script terminates with exit code "1" which must be caught by the deployment service to inform the user.
+# 4. The script invokes setup.sh script and checks for errors returned by the script, logging to a file in the VM.
+
+# Enable this for debugging only
+##set -ex
+
+APP_PATH="/app"
+SETUP_LOG="${APP_PATH}/setup.log"
+
+# Loop through arguments and extract some parameters, without modifying $@ needed later
+for X in "$@"; do
+    case "$PREVIOUS" in
+        --release-version) PCS_RELEASE_VERSION="$X" ;;
+    esac
+    PREVIOUS="$X"
+done
+
+if [ -z "$PCS_RELEASE_VERSION" ]; then
+    echo "Release version not specified (see --release-version)"
+    exit 1
+fi
```

Comment:
since you're using the ARM template to download `setup.sh` you can drop lines 18 to 29